### PR TITLE
k8s deploy

### PR DESCRIPTION
**Pull request by cmyui**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -100,11 +101,17 @@ func main() {
 		}
 	}
 
+	var tlsConfig *tls.Config
+	if settings.REDIS_USE_SSL {
+		tlsConfig = &tls.Config{}
+	}
+
 	// initialise redis
```
Comment (cmyui) [inline main.go:109]:
```suggestion
	// initialise redis
	var tlsConfig *tls.Config
	if settings.REDIS_USE_SSL {
		tlsConfig = &tls.Config{}
	}

```

File: web/templates/premium.html [hunk 2]
```diff
@@ -113,7 +113,7 @@ <h2 class="ui header">
                 <input
                   type="hidden"
                   name="custom"
-                  value='{"userid":"{{ $global.User.ID }}"}' />
+                  value="userid={{ $global.User.ID }}" />
```
Comment (cmyui) [inline web/templates/premium.html]: wtf why did i put this here

Comment (cmyui) [reply web/templates/premium.html]: highly based parsing code: https://github.com/osuAkatsuki/payments-service/blob/1d2823f42c78ea049bb5b63cc257c228642cf3e3/app/api/webhooks/paypal.py#L246

Comment (tsunyoku) [reply web/templates/premium.html]: what the fuck HAHA

**Pull request merged**